TWLight/resources/templates/resources/partner_filter.html

```diff
@@ -49,4 +49,8 @@ <h2>{% trans "Browse Partners" %}</h2>
       </div>
     {% endfor %}
   </div>
+
+  <a id="back-to-top" href="#" class="btn btn-light btn-sm">
```

Comment:
Can you add a tooltip that says `Back to top` along with its corresponding translator comment?

Reply:
added a tooltip :)